docs/tutorials/tut-design.md

```diff
@@ -0,0 +1,245 @@
+-  The SOC team wants to include a new source of Incidents in Cortex XSOAR from a security product that is currently not supported by the Marketplace. This could be already achieved through a SIEM, but a direct integration will make it easier to consume and provide a better UX to the analysts.
+- The SOC team wants Cortex XSOAR to automatically provide reputation information about IOCs from a Threat Intelligence source that is currently not supported by Cortex XSOAR.
+- The SOC team wants to integrate to an existing IT solution (i.e. a CMDB, Instant Messaging platform or Database) to automatically exchange data between with Cortex XSOAR.
+- The SOC team wants to automatically trigger actions on a third party security product that is currently not available in the Cortex XSOAR Marketplace.
+
+
+If you are a third-party security vendor and want to integrate your product, [here](docs/use-cases) are some common Use Cases across different product categories that you could use as a starting point.However,  don't limit your imagination to what is there, as your product has unique capabilities that could be exposed through Cortex XSOAR  to provide lots of value to our joint customers.
+
+Another important recommendation about this design phase, is not to necessarily focus on APIs when designing the use cases. Think about the outcomes and write them down first, then look at your APIs to understand if they can be achieved or if there are any gaps. During our design sessions with technology partners, we often found them deciding to implement new APIs in their platforms to improve the level of automation that customers could achieve.
+
+## Contribution Scope
+
+Now that you have an idea of what Use Cases you want to create, it's time to determine how they translate into a Cortex XSOAR contribution.
+
+All contributions consist in [Content Packs](../concepts/concepts#content-packs): the first think to understand is what you should create and add to your pack.
+
+Depending on whether you are a Palo Alto Networks Technology Partner, an Individual Contributor, a Customer or an Enthusiast and depending on how you want your contribution to appear on the [Cortex XSOAR Marketplace](../partners/marketplace), the thoroughness and complexity of your final product will be different.
+
+This section summarizes some guidelines to get started.
+
+### Playbooks
+
+If you have an end-to-end use case in mind, most likely you are going to build one or more **Playbooks**. Start thinking about the process that you want to automate and enumerate the steps and the decisions that are part of it: those will become your playbook's tasks. Figure out if all these building blocks are already part of Cortex XSOAR by browsing the Marketplace.
+
+You can have multiple Playbooks in the same Content Pack, as long as they are related to a similar end-to-end use case. If they are completely separate, consider splitting them in multiple Content Packs.
+

```

Comment:
add a few playbooks examples from the reference?
also, let's mention that we can use tasks from other packs as mandatory/non mandatory.
e.g: I'll do endpoint management: use XDR, CB, crowdStrike and cyclance packs in my pack.